fix: Load ace syntax hints when possible (with setTimeout)

redref opened the pull request:
ACE is loaded in async mode, this is the only way for now to "check" ace is loaded.

Fix syntax highlighting in Grafana 6.X context.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ovh/ovh-warp10-datasource?pullRequest=46) <br/>All committers have signed the CLA.

Commit:
fix: Load ace syntax hints when possible (with setTimeout)

ACE is loaded in async mode, this is the only way for now to "check" ace is loaded.
